packages/html/src/SimpleTable.ts

```diff
@@ -1,6 +1,9 @@
 import { HTMLWidget } from "@hpcc-js/common";
 import { select as d3Select } from "d3-selection";
 
+function entitiesEncode(str) {
```

Comment:
Missing typings

Reply:
Duplicate code with:  https://github.com/hpcc-systems/Visualization/blob/b6817413f1d7a7aef2ee9115c43d05b54f81fdb9/packages/eclwatch/src/WUResultStore.ts#L4-L20 (common up and move to util.